doc/ImplementingApplications/ConceptOfApplicationPatternPackage/ConceptOfApplicationPatternPackage.md

```diff
@@ -14,7 +14,13 @@ This library provides APIs,
 
```

Comment:
can we update the applicationPatternPackages.png according to the latest version

Reply:
Done.